packages/web/components/cards/pool-card.tsx

```diff
@@ -0,0 +1,197 @@
+    pool.id,
+    priceStore,
+    priceStore.getFiatCurrency("usd")!
+  );
+
+  const poolLiquidity = pool.computeTotalValueLocked(
+    priceStore,
+    priceStore.getFiatCurrency("usd")!
+  );
+
+  const bondedShareRatio =
+    queryOsmosis.queryGammPoolShare.getLockedGammShareRatio(
+      accountInfo.bech32Address,
+      pool.id
+    );
+  const bonded = poolLiquidity
+    .mul(bondedShareRatio.moveDecimalPointLeft(2))
+    .toString();
+
+  return (
+    <div
+      className="px-[1.875rem] pt-8 pb-6 bg-card rounded-2xl cursor-pointer hover:ring-1 hover:ring-enabledGold"
+      onClick={() => router.push(`/pools/${pool.id}`)}
+    >
+      <div className="relative flex items-center">
+        <div
+          className={
+            "absolute z-10 w-[4.125rem] h-[4.125rem] rounded-full border-[1px] bg-card border-enabledGold flex items-center justify-center"
+          }
+        >
+          {pool.poolAssets[0].amount.currency.coinImageUrl ? (
+            <Image
+              src={pool.poolAssets[0].amount.currency.coinImageUrl}
+              alt={pool.poolAssets[0].amount.currency.coinDenom}
+              width={54}
+              height={54}
+            />
+          ) : (
+            <div
+              className={classNames(
+                "w-[54px] h-[54px] bg-blue rounded-full flex items-center justify-center",
+                poolCardIconBackgroundColorsToTailwindBgImage[
+                  PoolCardIconBackgroundColors[
+                    deterministicInteger % PoolCardIconBackgroundColors.length
+                  ]
+                ]
+              )}
+            >
+              <Image
+                src="/icons/OSMO.svg"
+                alt="no token icon"
+                width={40}
+                height={40}
+              />
+            </div>
+          )}
+        </div>
+        <div
+          className={
+            "ml-10 mr-6 w-[4.125rem] h-[4.125rem] rounded-full border-[1px] border-enabledGold shrink-0 flex items-center justify-center"
+          }
+        >
+          {pool.poolAssets.length >= 3 ? (
+            <div className="body1 text-white-mid ml-2.5">{`+${
+              pool.poolAssets.length - 1
+            }`}</div>
+          ) : pool.poolAssets[1].amount.currency.coinImageUrl ? (
+            <Image
+              src={pool.poolAssets[1].amount.currency.coinImageUrl}
+              alt={pool.poolAssets[1].amount.currency.coinDenom}
+              width={52}
+              height={52}
+            />
+          ) : (
+            <div
+              className={classNames(
+                "w-[54px] h-[54px] bg-blue rounded-full flex items-center justify-center",
+                poolCardIconBackgroundColorsToTailwindBgImage[
+                  PoolCardIconBackgroundColors[
+                    deterministicInteger % PoolCardIconBackgroundColors.length
+                  ]
+                ]
+              )}
+            >
+              <Image
+                src="/icons/OSMO.svg"
```

Comment:
Instead of using `/icons/OSMO.svg`: in the old frontend we have a `/public/assets/common/missing.svg` that we could use here. This would not mislead users that the pool has OSMO just because an asset icon wouldn't load for some reason.